src/eko/evolution_operator/physical.py

```diff
@@ -75,6 +75,76 @@ def ad_to_evol_map(cls, op_members, nf, q2_final, intrinsic_range):
         # map key to MemberName
         return cls.promote_names(m, q2_final)
 
+    @classmethod
+    def ad_to_uni_evol_map(cls, op_members, nf, q2_final, intrinsic_range):
+        """
+        Obtain map between the 3-dimensional anomalous dimension basis and the
+        4-dimensional evolution basis.
+
+        .. todo:: in VFNS sometimes IC is irrelevant if nf>=4
+
+        Parameters
+        ----------
+            op_members : dict
+                operator members in anomalous dimension basis
+            nf : int
+                number of active light flavors
+            intrinsic_range : sequence
+                intrinsic heavy flavors
+
+        Returns
+        -------
+            m : dict
+                map
+        """
+        # constant elements
+        m = {
+            "g.g": op_members[(21, 21)],
+            "g.ph": op_members[(21, 22)],
+            "g.S": op_members[(21, 100)],
+            "g.Sdelta": op_members[(21, 101)],
+            "ph.g": op_members[(22, 21)],
+            "ph.ph": op_members[(22, 22)],
+            "ph.S": op_members[(22, 100)],
+            "ph.Sdelta": op_members[(22, 101)],
+            "S.g": op_members[(100, 21)],
+            "S.ph": op_members[(100, 22)],
+            "S.S": op_members[(100, 100)],
+            "S.Sdelta": op_members[(100, 101)],
+            "V.V": op_members[(10200, 10200)],
```

Comment:
Choose your convention: either NS stuff as 0 as second argument or you repeat the first label

Reply:
For V.V i didn't use the notation (pid, 0) because it belongs to the 2x2 sector (even if it is a nonsinglet). In fact we have also V.Vdelta and Vdelta.Vdelta

Reply:
you're right